src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -449,6 +460,11 @@ def parse_rpc_arg(rpc_arg: str) -> Sequence[str]:
         default=50052,
         type=int,
         help="The port on which to expose the peer distribution stats service.")
+    parser.add_argument(
+        "--secure_mode",
+        default="False",
+        type=str,
```

Comment:
This can also be a bool.

Reply:
I don't think that behavior is desirable. `argparse` just treats the type as a callable and uses it to deserialize the argument:

```
(venv) rbellevi@psm-security-interop-test-dev:~/grpc$ python
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> bool("true")
True
>>> bool("false")
True
>>> bool("False")
True
```

Reply:
Resolved with more robust bool parsing.